azure-devops/install_msvc_preview.ps1

```diff
@@ -0,0 +1,65 @@
+$ErrorActionPreference = "Stop"
+
+Function InstallVS
+{
+    # Microsoft hosted agents do not have the required MSVC 14.23 for building MSVC STL. 
```

Comment:
We should say "MSVC Preview" instead of "MSVC 14.23" so this comment won't be forever out-of-date. (And the next line.)